examples/reference-entity-model/src/main/ml-config/security/users/hub-developer.json

```diff
@@ -8,6 +8,7 @@
     "hub-central-entity-model-writer",
     "hub-central-load-writer",
     "hub-central-mapping-writer",
+    "hub-central-custom-reader",
```

Comment:
Ryan's PR just got merged and hub-central-operator inherits hub-central-custom-reader and so available to hub-central-developer as well.